src/pages/TeamSettings/Service-Accounts-Table.vue

```diff
@@ -306,11 +307,9 @@ export default {
             >
               <v-list-item-title>{{ key.name }}</v-list-item-title>
               <v-list-item-subtitle
-                >Created
-                {{
-                  key.created_at ? formDate(key.created_at) : ''
-                }}</v-list-item-subtitle
-              >
+                >Created {{ key.created_at ? formDate(key.created_at) : '' }}
+                {{ key.created_by ? ' by '.concat(key.created_by) : '' }}
```

Comment:
This works but might be simpler/neater code to use a span and v-if
```suggestion
                <span v-if="key.created_by">by {{ key.created_by }}</span>
```